src/test/java/com/uber/jenkins/phabricator/PhabricatorNotifierTest.java

```diff
@@ -46,7 +46,12 @@ public void setUp() throws IOException {
                 ".phabricator-comment",
                 "1001",
                 false,
-                true
+                true,
+                true,
+                ".phabricator-lint",
+                "10000",
+                ".phabricator-warning",
```

Comment:
Why do we have a file for lint and a file for warnings? I think this should just be a single file, and you can set the **severity**, which could be advice, warning, error, etc.